### PR TITLE
New typos from LO

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5577,6 +5580,7 @@ hiddden->hidden
 hidded->hidden
 hiddin->hidden, hiding,
 hidding->hiding, hidden,
+hided->hidden
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "hid"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9839,6 +9844,7 @@ retalitation->retaliation
 retangles->rectangles
 retcieves->retrieves, receives,
 retet->reset, retest,
+retetting->resetting
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `retesting`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -2502,6 +2502,7 @@ conent->content
 conents->contents
 conesencus->consensus
 conetxt->context
+conext->context
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Also `connect, connects,`

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

**Pull request merged**